### ISSUE TITLE
Revised legend specs

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/wc-gocam-viz#59]
-direct positive regulation/activation
-direct negative regulation/inhibition
-indirect positive regulation
-indirect negative regulation

-provides input for
-removes input for
-input of
-has output

-constitutively upstream
-upstream positive effect
-upstream negative effect

@tmushayahama - playing around with different possible legends to the gocam-viz, I'm thinking that a list like this with the corresponding colored glyph to the left of each text line might be simplest.  We can look at this together next week.

[spreadsheet](https://docs.google.com/spreadsheets/d/1Gt1rgPocGQ4uNKgzBhbp3vCJa-HRzzjPx0GQkwcj81Q/edit?gid=1769545376#gid=1769545376) of relations and glyphs

**Comment by tmushayahama** [comment]
@vanaukenk does this mean the Direct Regulation and indirect Regulation gray lines will be removed on the legend

**Comment by vanaukenk** [comment]
@tmushayahama 

Yes, that way there's a clearer distinction between indirect regulation (when we know the mechanism) and causally upstream of, when we don't.
